everest

```diff
@@ -564,33 +564,43 @@ OCAML
   #
   # regtool get \HKLM\Software\Python\PythonCore\2.7\InstallPath\
   # regtool get \HKCU\Software\Python\PythonCore\2.7\InstallPath\
+  # CMW: The following are the 32-bit installations, I think.
```

Comment:
can we update / remove the comment that says "what we really should do"

Reply:
Sure, will do!